rust_snuba/src/consumer.rs

```diff
@@ -249,47 +245,33 @@ pub fn consumer_impl(
         rebalancing::delay_kafka_rebalance(secs)
     }
 
-    let processor = if mutations_mode {
-        let mut_factory = MutConsumerStrategyFactory {
-            storage_config: first_storage,
-            max_batch_size,
-            max_batch_time,
-            processing_concurrency: ConcurrencyConfig::new(concurrency),
-            clickhouse_concurrency: ConcurrencyConfig::new(clickhouse_concurrency),
-            health_check_file: health_check_file.map(ToOwned::to_owned),
-            batch_write_timeout,
-        };
-
-        StreamProcessor::with_kafka(config, mut_factory, topic, dlq_policy)
```

Comment:
Is the mutations consumer idea gone for good ?

Reply:
yes it was pretty much abandoned a while ago. IIUC we got most of its needs covered by span buffer